toolkit/tools/imager/imager.go

```diff
@@ -127,12 +127,11 @@ func buildSystemConfig(systemConfig configuration.SystemConfig, disks []configur
 		defaultTempDiskName   = "disk.raw"
 		existingChrootDir     = false
 		leaveChrootOnDisk     = false
-		marinerReleasePackage = "mariner-release"
 		grub2Package          = "grub2"
+		marinerReleasePackage = "azurelinux-release"
```

Comment:
Nit: This variable made me laugh a bit. Thinking about it this seems quite brittle but not your problem.
```suggestion
		distroReleasePackage = "azurelinux-release"
```